fuzz/src/main/java/edu/berkeley/cs/jqf/fuzz/ei/ZestCLI.java

```diff
@@ -63,7 +73,7 @@
             description = "Output Directory containing results (default: fuzz_results)")
     private File outputDirectory = new File("fuzz-results");
 
-    @Parameters(index="0", paramLabel = "PACKAGE", description = "package containing the fuzzer and all dependencies")
+    @Parameters(index = "0", paramLabel = "PACKAGE", description = "package containing the fuzzer and all dependencies")
```

Comment:
Should this be "package containing the fuzz target and all dependencies"? Isn't the fuzzer in the JAR that runs ZestCLI itself?

Reply:
yeah forgot to fix this one.